### DISCUSSION TITLE
Hosting a Milvus Database

### DISCUSSION THREAD

**Discussion by KellenMossner** [milvus-io/milvus#36193] [Q&A and General discussion] [answered]
I need to host a milvus database preferably on GCP and a standalone database. We are a small team and need to individually access the database remotely. I have tried setting up a standalone milvus database on Google Kubernetes Engine with LoadBalancer but I am never able to connect to the external IP. Please can someone assist me with this or give me a guide to follow, I am still very new to Milvus.

**Comment by yhmo** [comment]
Did you follow this doc? https://milvus.io/docs/gcp.md#Deploy-a-Milvus-Cluster-on-GKE

**Reply by KellenMossner** [reply]
Yes, followed it exactly but cannot connect to the external ip and port

**Reply by yhmo** [reply]
Wait @LoveEachDay comment.

**Reply by LoveEachDay** [reply, marked as answer]
@KellenMossner Here's the procedures that help to resolve the connection issue:
1. Try to use `kubectl port-forward svc/<milvus-svc> 19530:19530` and access the `localhost:19530` to verify whether the deployment of milvus has been successfully setup.
2. If step 1 is success, we'd check that the external ip of the LoadBalancer milvus svc can be accessed, use nc to check `nc -zv <milvus-svc-lb-ip> 19530`; if step 1 is failed, check whether all the pods are running, upload all the logs using the script: https://github.com/milvus-io/milvus/blob/master/deployments/export-log/export-milvus-log.sh
3. if step 2 is failed, try to check the firewall or network policies
